ERP/resources/views/components/user-type-dropdown.blade.php

```diff
@@ -0,0 +1,11 @@
+
+<select id="user_type" name="user_type" class="form-control" required>
+    <option value="">-- SELECT USER TYPE --</option>
+    <option value="0" @if(isset($user) && $user->user_type ==0) selected @endif>IT Department</option>
+    <option value="1" @if(isset($user) && $user->user_type ==1) selected @endif>Human Resources (HR)</option>
```

Comment:
I think we should remove HR and Floor Worker?

Reply:
I removed them. However, since we already used `5` to define Manufacturer worker, I didn't change it to `2` (since now there is no user associated with `user_type=1` or `user_type=2`

If i DID change Manufacturer worker to `user_type=2`, then i would need to change it in multiple places like where we limit user access.

- So, basically, we can add/remove user types without issues but we can NOT modify a user_type. Once we define it, we can not modify it to a different integer. We can only remove it, and, once we remove it, we can not assign another user type the same integer value.